guides/using-lastpass.md

```diff
@@ -4,50 +4,101 @@ expires: 2019-09-01
 ---
 # Using LastPass Enterprise
 
-## The product
+## What is LastPass?
 
-LastPass is an online password management tool that we make available to you to help you store and share passwords. Using it means you no longer need to remember dozens of passwords, just a single master password. It keeps all your website logins protected, helps with creating new passwords, password sharing where required, and ensuring passwords are suitably strong.
+LastPass is an online password management tool that we make available to you to help you create, store and share passwords. Using it means you no longer need to remember dozens of passwords, just a single master password. It keeps all your website logins protected, helps with creating new ‘strong’ passwords and passwording sharing when required.
 
-The LastPass software will be installed on your browsers as a browser extension. As you go to work-related sites, save the credentials to your LastPass vault. LastPass will then autofill those passwords the next time you go to log in.
+LastPass is available as a browser extension for popular browsers and as well as a full software suite (for use outside of browsers) for Microsoft Windows and Apple macOS.
```

Comment:
Something about 'contact service desk to get the extension installed'

Reply:
I think we need @SteveMarshall 's view on 'support'

Browser extension:
I don't _think_ there are any restrictions set by G-Suite/Google policy by DSD that stop people self-installing the browser extension. No idea about on IE/Firefox on DOM1.

'Fat' Windows app:
That needs Atos to wrap it into SelfService?

'Fat' macOS app (https://apps.apple.com/us/app/lastpass/id926036361?ls=1&mt=12):
I again don't know MDM controls for MOJDT macOS devices. Can people self install this? If so, which Apple ID do they use for the 'purchase' (etc)

Reply:
bump @SteveMarshall

Reply:
@joelgsamuel I'm honestly not sure what you need from me here? Personally, I don't care which route… And I'm not sure I have any way to decide which would work, either?

Reply:
@SteveMarshall if we're suggesting fat clients, people need to be able to install that (admin rights in local OS likely required) which needs some collaboration with DSD. I guess we're looking for steer on what you are organising with them, or not.

(I think we prefer fat clients)

Reply:
I'm organising nothing; I'm not being responsible for rolling it out formally across the organisation, just making licenses available for people who need infrastructure.